interactions:
- request:
    body: 'request={''ruleset_name'': ''nonexistent''}'
    headers:
      Connection:
      - close
      Content-Length:
      - '39'
      Content-Type:
      - application/x-www-form-urlencoded
      Host:
      - localhost
      User-Agent:
      - Python-urllib/3.5
    method: POST
    uri: http://localhost/checkmd2/check_mk/webapi.py?output_format=python&_secret=%40HEBECXCDWXXISGBAAWQ&_username=automation&request_format=python&action=get_ruleset
  response:
    body:
      string: '{''result'': ''Unhandled exception: Traceback (most recent call last):\n  File
        "/omd/sites/checkmd2/share/check_mk/web/htdocs/webapi.py", line 251, in page_api\n    action_response
        = api_actions[action]["handler"](request_object)\n  File "/omd/sites/checkmd2/share/check_mk/web/plugins/webapi/webapi.py",
        line 604, in _get\n    ruleset_dict = self._get_ruleset_configuration(ruleset_name)\n  File
        "/omd/sites/checkmd2/share/check_mk/web/plugins/webapi/webapi.py", line 586,
        in _get_ruleset_configuration\n    ruleset = collection.get(ruleset_name)\n  File
        "/omd/sites/checkmd2/share/check_mk/web/htdocs/watolib.py", line 7813, in
        get\n    return self._rulesets[name]\nKeyError: \''nonexistent\''\n'', ''result_code'':
        1}'
    headers:
      Cache-Control:
      - no-cache
      Connection:
      - close
      Content-Length:
      - '712'
      Content-Type:
      - text/plain; charset=UTF-8
      Date:
      - Fri, 22 Mar 2019 15:32:10 GMT
      Server:
      - Apache/2.4.25 (Debian)
      Vary:
      - Cookie,Accept-Encoding
    status:
      code: 200
      message: OK
- request:
    body: 'request={''ruleset_name'': ''nonexistent''}'
    headers:
      Connection:
      - close
      Content-Length:
      - '39'
      Content-Type:
      - application/x-www-form-urlencoded
      Host:
      - localhost
      User-Agent:
      - Python-urllib/3.5
    method: POST
    uri: http://localhost/checkmd2/check_mk/webapi.py?output_format=python&_secret=%40HEBECXCDWXXISGBAAWQ&_username=automation&action=get_ruleset&request_format=python
  response:
    body:
      string: '{''result'': ''Unhandled exception: Traceback (most recent call last):\n  File
        "/omd/sites/checkmd2/share/check_mk/web/htdocs/webapi.py", line 251, in page_api\n    action_response
        = api_actions[action]["handler"](request_object)\n  File "/omd/sites/checkmd2/share/check_mk/web/plugins/webapi/webapi.py",
        line 604, in _get\n    ruleset_dict = self._get_ruleset_configuration(ruleset_name)\n  File
        "/omd/sites/checkmd2/share/check_mk/web/plugins/webapi/webapi.py", line 586,
        in _get_ruleset_configuration\n    ruleset = collection.get(ruleset_name)\n  File
        "/omd/sites/checkmd2/share/check_mk/web/htdocs/watolib.py", line 7813, in
        get\n    return self._rulesets[name]\nKeyError: \''nonexistent\''\n'', ''result_code'':
        1}'
    headers:
      Cache-Control:
      - no-cache
      Connection:
      - close
      Content-Length:
      - '712'
      Content-Type:
      - text/plain; charset=UTF-8
      Date:
      - Fri, 22 Mar 2019 15:34:13 GMT
      Server:
      - Apache/2.4.25 (Debian)
      Vary:
      - Cookie,Accept-Encoding
    status:
      code: 200
      message: OK
- request:
    body: 'request={''ruleset_name'': ''nonexistent''}'
    headers:
      Connection:
      - close
      Content-Length:
      - '39'
      Content-Type:
      - application/x-www-form-urlencoded
      Host:
      - localhost
      User-Agent:
      - Python-urllib/3.5
    method: POST
    uri: http://localhost/checkmd2/check_mk/webapi.py?action=get_ruleset&output_format=python&_secret=%40HEBECXCDWXXISGBAAWQ&request_format=python&_username=automation
  response:
    body:
      string: '{''result'': ''Unhandled exception: Traceback (most recent call last):\n  File
        "/omd/sites/checkmd2/share/check_mk/web/htdocs/webapi.py", line 251, in page_api\n    action_response
        = api_actions[action]["handler"](request_object)\n  File "/omd/sites/checkmd2/share/check_mk/web/plugins/webapi/webapi.py",
        line 604, in _get\n    ruleset_dict = self._get_ruleset_configuration(ruleset_name)\n  File
        "/omd/sites/checkmd2/share/check_mk/web/plugins/webapi/webapi.py", line 586,
        in _get_ruleset_configuration\n    ruleset = collection.get(ruleset_name)\n  File
        "/omd/sites/checkmd2/share/check_mk/web/htdocs/watolib.py", line 7813, in
        get\n    return self._rulesets[name]\nKeyError: \''nonexistent\''\n'', ''result_code'':
        1}'
    headers:
      Cache-Control:
      - no-cache
      Connection:
      - close
      Content-Length:
      - '712'
      Content-Type:
      - text/plain; charset=UTF-8
      Date:
      - Fri, 22 Mar 2019 15:37:32 GMT
      Server:
      - Apache/2.4.25 (Debian)
      Vary:
      - Cookie,Accept-Encoding
    status:
      code: 200
      message: OK
- request:
    body: 'request={''ruleset_name'': ''nonexistent''}'
    headers:
      Connection:
      - close
      Content-Length:
      - '39'
      Content-Type:
      - application/x-www-form-urlencoded
      Host:
      - localhost
      User-Agent:
      - Python-urllib/3.5
    method: POST
    uri: http://localhost/checkmd2/check_mk/webapi.py?_secret=2396a766-8de5-4951-b2b6-8b81fc946294&output_format=python&_username=automation&action=get_ruleset&request_format=python
  response:
    body:
      string: '{''result'': ''Unhandled exception: Traceback (most recent call last):\n  File
        "/omd/sites/checkmd2/share/check_mk/web/htdocs/webapi.py", line 251, in page_api\n    action_response
        = api_actions[action]["handler"](request_object)\n  File "/omd/sites/checkmd2/share/check_mk/web/plugins/webapi/webapi.py",
        line 604, in _get\n    ruleset_dict = self._get_ruleset_configuration(ruleset_name)\n  File
        "/omd/sites/checkmd2/share/check_mk/web/plugins/webapi/webapi.py", line 586,
        in _get_ruleset_configuration\n    ruleset = collection.get(ruleset_name)\n  File
        "/omd/sites/checkmd2/share/check_mk/web/htdocs/watolib.py", line 7813, in
        get\n    return self._rulesets[name]\nKeyError: \''nonexistent\''\n'', ''result_code'':
        1}'
    headers:
      Cache-Control:
      - no-cache
      Connection:
      - close
      Content-Length:
      - '712'
      Content-Type:
      - text/plain; charset=UTF-8
      Date:
      - Mon, 25 Mar 2019 16:56:56 GMT
      Server:
      - Apache/2.4.25 (Debian)
      Set-Cookie:
      - language=en; path=/; httponly
      Vary:
      - Cookie,Accept-Encoding
    status:
      code: 200
      message: OK
version: 1
